plugins/autowarmth.koplugin/suntime.lua

```diff
@@ -291,11 +439,11 @@ function SunTime:calculateTimes()
     self.times[8]  = self.set_civil
     self.times[9]  = self.set_nautic
     self.times[10] = self.set_astronomic
-    self.times[11] = self.noon + 12
+    self.times[11] = self.midnight
 end
 
--- get time in seconds (either actual time in hours or date struct)
-function SunTime:getTimeInSec(val)
+-- Get time in seconds (either actual time in hours or date struct)
+function SunTime:getTimeInSec(val) -- luacheck: ignore 212
```

Comment:
This plugin is your domain of course, but we might ask why this ignore is here for example.

Reply:
Thank you. This is a leftover from an old test script.
I will remove that on the next PR.